vsintegration/src/FSharp.Editor/Common/RoslynHelpers.fs

```diff
@@ -187,19 +187,30 @@ module internal OpenDeclarationHelper =
 
         let getLineStr line = sourceText.Lines.[line].ToString().Trim()
         let pos = ParsedInput.adjustInsertionPoint getLineStr ctx
-        let docLine = pos.Line - 1
+        let docLine = Line.toZ pos.Line
         let lineStr = (String.replicate pos.Column " ") + "open " + ns
-        let sourceText = sourceText |> insert docLine lineStr
+
+        // If we're at the top of a file (e.g., F# script) then add a newline before adding the open declaration
+        let sourceText =
+            if docLine = 0 then
+                sourceText
+                |> insert docLine Environment.NewLine
```

Comment:
Why are we inserting a newline?

Reply:
Imagine you type `ase` and select `Enumerable.AsEnumerable` from the list.

Without the newline, this will change the text to `open System.LinqEnumerable.AsEnumerable`.

The code fix also inserts newlines when inserting an open declaration to an existing set of them like so:

Before:
```fsharp
open System
File // Invoke code fix to open System.IO
```

After:
```fsharp
open System
open System.IO

File
```

So this keeps that consistent.